tc-health-client/build/build_rpm.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env sh
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# shellcheck shell=ash
+trap 'exit_code=$?; [ $exit_code -ne 0 ] && echo "Error on line ${LINENO} of ${0}" >/dev/stderr; exit $exit_code' EXIT;
+set -o errexit -o nounset -o pipefail -o xtrace;
+
+#----------------------------------------
+importFunctions() {
+	local script scriptdir;
+	script="$(realpath "$0")";
+	scriptdir="$(dirname "$script")";
+	HC_DIR="$(dirname "$scriptdir")";
+	TC_DIR="$(dirname "$HC_DIR")";
+	export HC_DIR TC_DIR;
+	functions_sh="$TC_DIR/build/functions.sh";
+	if [ ! -r "$functions_sh" ]; then
+		echo "error: can't find $functions_sh" >&2;
+		return 1;
+	fi
+	. "$functions_sh";
+}
+
+#----------------------------------------
+initBuildArea() {
+	echo "Initializing the build area for t3c";
```

Comment:
The build area is not being initialized for t3c

Reply:
@zrhoffman changed